darts/tests/models/forecasting/test_torch_forecasting_model.py

```diff
@@ -290,6 +291,74 @@ def test_train_from_10_n_epochs_20_fit_15_epochs(self):
             model1.fit(self.series, epochs=15)
             self.assertEqual(15, model1.epochs_trained)
 
+        def test_load_weights_from_checkpoint(self):
+            original_model_name = "original"
+            retrained_model_name = "retrained"
+            # original model, checkpoints are saved
+            model1 = RNNModel(
+                12,
+                "RNN",
+                5,
+                1,
+                n_epochs=5,
+                work_dir=self.temp_work_dir,
+                save_checkpoints=True,
+                model_name=original_model_name,
+            )
+            model1.fit(self.series[:90])
+            original_preds = model1.predict(10)
+            original_mape = mape(original_preds, self.series[90:])
+
+            # load last checkpoint of original model, train it for 2 additional epochs
+            model_rt = RNNModel(
+                12,
+                "RNN",
+                5,
+                1,
+                n_epochs=5,
+                work_dir=self.temp_work_dir,
+                model_name=retrained_model_name,
+            )
+            model_rt.load_weights_from_checkpoint(
```

Comment:
It would be good to add a check here that `model_rt` (before fitting) produces identical forecasts as `model1`